freeipa/src/main/java/com/sequenceiq/freeipa/entity/FreeIpa.java

```diff
@@ -92,4 +94,16 @@ public void setAdminPassword(String adminPassword) {
     public String getAccountId() {
         return stack.getAccountId();
     }
+
+    @Override
+    public String toString() {
+        return new StringJoiner(", ", FreeIpa.class.getSimpleName() + "[", "]")
+                .add("id=" + id)
+                .add("stack='" + stack + '\'')
+                .add("hostname='" + hostname + "'")
+                .add("domain='" + domain + "'")
+                .add("adminGroupName='" + adminGroupName + "'")
+                .add("adminPassword='***'")
```

Comment:
I think it would be easier just to omit these

Reply:
I wasn't sure if you wanted me to omit all of them or just the masked adminPassword. I removed the masked adminPassword, but I can remove the entire toString() if you think that has benefits.

Reply:
I meant the password, thanks. I mean if we mask it, then it doesn't make much sense to log it, because it could be null.
Others make sense, so I'm always glad when `toString` is implemented and updated. Thanks!